tools/install_demo_configuration.bat

```diff
@@ -324,7 +324,7 @@ echo plugins.security.system_indices.indices: [".plugins-ml-model", ".plugins-ml
 	if %cluster_mode% == 1 (
         echo network.host: 0.0.0.0 >> "%OPENSEARCH_CONF_FILE%"
         echo node.name: smoketestnode >> "%OPENSEARCH_CONF_FILE%"
-        echo cluster.initial_master_nodes: smoketestnode >> "%OPENSEARCH_CONF_FILE%"
```

Comment:
Can you remove the changes from this file?

Reply:
Yeah I just brought it back but I can drop it.